R/doAzureParallel.R

```diff
@@ -548,8 +550,6 @@ setHttpTraffic <- function(value = FALSE) {
     return(taskId)
   })
 
-  rAzureBatch::updateJob(id)
-
   if (enableCloudCombine) {
     cat("\nSubmitting merge task")
     mergeTaskId <- paste0(id, "-merge")
```

Comment:
just keep this as 'merge'? that way when searching we don't need to know the prefix by number.